app/services/meta-tags.ts

```diff
@@ -53,10 +53,15 @@ export interface LinkMetaTagAttrs {
     href: string;
 }
 
-export type MetaTagAttrs = NameMetaTagAttrs | PropMetaTagAttrs | LinkMetaTagAttrs;
+export interface ScriptTagAttrs {
+    type: Content;
+}
+
+export type MetaTagAttrs = NameMetaTagAttrs | PropMetaTagAttrs | LinkMetaTagAttrs | ScriptTagAttrs;
 
 export interface HeadTagDef {
     type: string;
+    content?: Content;
```

Comment:
`content` is also a `string`.

Reply:
As per your review, I've updated the type separately to prevent issues in the pre-defined meta-tags service.